resources/views/battle-results.blade.php

```diff
@@ -68,4 +69,17 @@
             margin-bottom: 10px;
         }
     </style>
-@endsection
\ No newline at end of file
+@endsection
+
+@section("scripts")
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jQuery-slimScroll/1.3.8/jquery.slimscroll.min.js"></script>
```

Comment:
Could you insert this lib as a npm dependency?

Also, i think this one is a little bit more popular:

https://github.com/malihu/malihu-custom-scrollbar-plugin